projects/04/mult/Mult.asm

```diff
@@ -9,4 +9,26 @@
 // This program only needs to handle arguments that satisfy
 // R0 >= 0, R1 >= 0, and R0*R1 < 32768.
 
-// Put your code here.
\ No newline at end of file
+@2 // set index 2 to 0
+M=0
+
+(LOOP)
+@0
+D=M
+
+@END
+D;JEQ // if index 0 is < 0 then end loop
+
+@1
+D=M
+@2
+M=M+D // put sum of index 2 and index 1 back to index 2

```

Comment:
re: my virtual register point, the comment for this chunk of four lines could be (for example) `// R2 = R2 + R1`.